### ISSUE TITLE
CLI: replace hyperlane-deploy with CLI for PI experience on v3

### ISSUE THREAD

**Issue by tkporter** [hyperlane-xyz/hyperlane-monorepo#2811]
Blocked by main being merged into v3, which Yorke is doing atm. This is intended to be done prior to any work on getting the CLI to work with the new registry format

- [ ] update the CLI to work with v3
- [ ] update docs
